presto-main/src/main/java/io/prestosql/server/PrestoServer.java

```diff
@@ -114,6 +115,9 @@ public void run()
         try {
             Injector injector = app.strictConfig().initialize();
 
+            log.info("Working directory: %s", Paths.get(".").toAbsolutePath().toRealPath());
+            log.info("Etc directory: %s", Paths.get("etc").toAbsolutePath().toRealPath());
```

Comment:
This seems redundant with the above (unless you really care about the target of a symlink).

Reply:
Actually, actual location of etc (target of `./etc`) is the thing which i care about.